data/impl/C_Texture.GetAtlasInfo.lua

```diff
@@ -0,0 +1,23 @@
+local uitextureatlas, uitextureatlasmember, atlas = ...
+local band = require('bit').band
+atlas = atlas:lower()
+for m in uitextureatlasmember() do
+  if m.CommittedName:lower() == atlas then
+    -- TODO use lookup by ID
+    for a in uitextureatlas() do
+      if a.ID == m.UiTextureAtlasID then
+        return {
+          bottomTexCoord = m.CommittedBottom,
+          file = a.FileDataID,
+          height = a.AtlasHeight,
+          leftTexCoord = m.CommittedLeft,
+          rightTexCoord = m.CommittedRight,
+          tilesHorizontally = not not band(m.CommittedFlags, 0x4),
+          tilesVertically = not not band(m.CommittedFlags, 0x2),
+          topTexCoord = m.CommittedTop,
+          width = a.AtlasWidth,
```

Comment:
Most of this is incorrect.

The width and height values are meant to be the size of the atlas _member_. This maps to the `OverrideWidth` and `OverrideHeight` columns if their values are non-zero, otherwise can be inferred by `CommittedRight - CommittedLeft` and `CommittedBottom - CommittedTop`.

The texcoords are also meant to be normalized - so divide each edge by the atlas width or height as needed. 

Finally, your tiling keys will always be true because even if the flags aren't set, `band` will return zero which is truthy. Just compare inequality to zero and it'll work.

[My little hacked up powershell script](https://gist.github.com/Meorawr/b92d0e1ca2ac09afc7b8440ac126987c) for generating a [TAV](https://www.curseforge.com/wow/addons/textureatlasviewer)-compatible atlas list should help.

Reply:
Thanks for all this. Should be fixed.
Note that your powershell script appears to be looking up override values from the atlas, not the member.